src/actions/getPriceOfNetworkDomain.ts

```diff
@@ -0,0 +1,16 @@
+import { getLogger } from "../utilities";
+import { ethers } from "ethers";
+import { DomainPurchaser } from "../contracts/types/DomainPurchaser";
+
+const logger = getLogger("actions:getDomainMetadata");
+
+export const getPriceOfNetworkDomain = async (
+  name: string,
+  domainPurchaser: DomainPurchaser
+): Promise<string> => {
+    logger.trace(`Get price of network domain for: ${name}`);
+    const tx = await domainPurchaser
```

Comment:
`tx` is commonly used for "transaction" while this would be more appropriate to call it `price` since you're queyring for the price.